app/content_type_views/core_super_feature_type.html

```diff
@@ -0,0 +1 @@
+GARBAGE
```

Comment:
<img src=http://67.media.tumblr.com/1c66ddc6447775bad1ae671c5089bd95/tumblr_mkdgfh79kU1rjxj9ko1_250.gif>

Real excited about what's going to happen here soon.